CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 
 ## [Unreleased]
 
+### Changed
+
+- Bump `stackable-operator` to 0.70.0 ([#267]).
```

Comment:
```suggestion
- Bump `stackable-operator` to 0.70.0, and other dependencies ([#267]).
```
Not sure if product-config needs a separate mention, but if not it's incorporated with this as well.

Reply:
There is no `product-config` dependency in the commons operator (it's a bit different since it isn't a product specific operator).

Reply:
Ah, I saw it here and assumed https://github.com/stackabletech/commons-operator/pull/267/files#diff-ac358776bc52d3fe028a6aece6b81a472e30226748e420de1dca6de541028020L4 okay then! :smile: